### ISSUE TITLE
Item turned into another item

### ISSUE THREAD

**Issue by Triadninja** [project-rubika/issues#1017]
### Reproduction steps

```bash
1. I equipped an Omni-Armed Forces Assault pack while in the subway
2. I killed a Looter in the subway, and looted one of the "Daily Tokens" that give sided tokens
3. The space that the Assault Pack was in before I equipped it was the place that the token was looted to in my inventory
4. Token icon in inventory remained, but had the properties of the assault pack
```

### Expected results

```bash
I would loot the token, and then right-click it to add it to my side tokens
```

### Actual results

```bash
The token turned into the Assault Pack, including it's functionality as a backpack, so when I right clicked it then the backpack would open, rather than consume the token.
```

### Bug prevalence

Only once

### Description

```bash
Aside from killing mobs/looting, nothing else of note. Relogging did fix the issue however.
```

### Report link

https://pastebin.com/HfTeNfB5

**Comment by RaggyLP** [comment]
Duplicate of #844.
